css/style.css

```diff
@@ -0,0 +1,9 @@
+#image {
+  width: 80%;
+}
+
+@media screen and(max-width: 720px) {
+  #image {
+    width: 60%;
```

Comment:
- [ ] [REQUIRED] Well done on this review comment 🙌🏻, however it is not clearly communicated and can not easily be understood by the person you are reviewing.
> A good comment includes a polite suggestion to an issue they found in code, it is then followed by an explanation why this change should be made.